tests/e2e/subscription_suite_test.go

```diff
@@ -19,7 +19,8 @@ var _ = Describe("Subscription Config Suite Test", func() {
 		err := dpaCR.Build(CSI)
 		Expect(err).NotTo(HaveOccurred())
 		//also test restic
-		dpaCR.CustomResource.Spec.Configuration.Restic.Enable = pointer.BoolPtr(true)
+		dpaCR.CustomResource.Spec.Configuration.NodeAgent.Enable = pointer.BoolPtr(true)
+		dpaCR.CustomResource.Spec.Configuration.NodeAgent.UploaderType = "restic"
```

Comment:
I'd add a case here for node-agent uploaderType kopia for another case.

Reply:
We can do, but I think it is https://issues.redhat.com/browse/OADP-2746 responsibility (here I made the change because our DPA validation does not allow new and old syntax together)

Reply:
cool.

Reply:
Note that uploaderType is irrelevant for CSI snapshot/datamover cases. The config value is ignored there. It's only ever used for fsbackup volumes.